docs/UserGuide.md

```diff
@@ -305,61 +305,81 @@ List all the events stored in the database with their unique index number.
 
 Delete the task with the unique index of `00123`.
 
-### Edit a floating task: `edit`
+### Edit a floating task/event/dealine: `edit`
+
+* Edit command can only edit the parameters of each commands but cannot transform a task to another
+type. For example, `eidt` cannot transform a floating task to a event task.
+
+#### Edit an floating task:
 
 Edit a floating task to revise its name or priority.
 
-    edit "TASK_NAME" [n-NEW_NAME | p-PRIORITY]...
+    edit TASK_UNIQUE_INDEX [n-NEW_NAME | p-PRIORITY]...
 
-* Task name should be in a pair of quotation marks.
+* Quotation marks are not necessary for `NEW_NAME`.
 
 * `PRIORITY` should only be the interger ranges from `0` to `5`.
 
-* Here we assume that no duplicate task name is used among all the task.
-
 #### Examples
 
-* `edit "washing clothes" p-0`
+* `edit 00124 p-0`
```

Comment:
`00124` is a lot of digits to type :scream: If I get one digit wrong, then I edit the wrong task and will have to undo and type the edit command again. Each extra digit I type is an extra chance to get it wrong.

Why not just keep the current behaviour where the index is based on the position of the task in the list in the user interface?